src/emulator/functionsEmulator.ts

```diff
@@ -253,30 +245,35 @@ export class FunctionsEmulator implements EmulatorInstance {
       await runtime.exit;
     };
 
-    hub.get(functionRoutes, handler);
-    hub.post(functionRoutes, handler);
+    hub.get(httpsFunctionRoutes, httpsHandler);
+    hub.post(httpsFunctionRoutes, httpsHandler);
+    hub.post(backgroundFunctionRoute, backgroundHandler);
 
-    this.server = hub.listen(this.port);
+    return hub;
   }
 
-  startFunctionRuntime(triggerName: string, proto?: any): FunctionsRuntimeInstance {
+  static startFunctionRuntime(
+    bundleTemplate: FunctionsRuntimeBundle,
+    triggerId: string,
+    nodeBinary: string,
+    proto?: any,
+    runtimeOpts?: InvokeRuntimeOpts
+  ): FunctionsRuntimeInstance {
     const runtimeBundle: FunctionsRuntimeBundle = {
+      ...bundleTemplate,
```

Comment:
So it looks likely that `triggerId` is double-specified here.  Which one will win?

Reply:
The later one will always win. So anything we specify explicitly here will overwrite the template and things left off (like disabled_features) will remain.